packages/components/src/pagination/documentation/usage-guidelines/usage.mdx

```diff
@@ -17,10 +17,10 @@ import Methods from './methods/usage.methods.mdx';
 
 ### Total Items
```

Comment:
I think we should mention "Default items per page"here in the title with "Total items" since they work together